examples/src/test/java/jdocs/cassandra/CassandraProjectionDocExample.java

```diff
@@ -237,9 +237,8 @@ public IllustrateRunningWithShardedDaemon() {
           ProjectionBehavior.Command.class,
           "shopping-carts",
           ShoppingCart.tags.size(),
-          id -> ProjectionBehavior.create(projection(ShoppingCart.tags.get(id))),
-          ShardedDaemonProcessSettings.create(system),
-          Optional.of(ProjectionBehavior.stopMessage())
+          id -> ProjectionBehavior.create(projection(ShoppingCart.tags.get((Integer) id))),
```

Comment:
Java is inferring `id` as `Object` hence the cast `(Integer)`. 

See also https://github.com/akka/akka/pull/29235